### DISCUSSION TITLE
Vertico TAB complete behavior: Paging candidates rather than choosing highlighted option?

### DISCUSSION THREAD

**Discussion by memestreak** [minad/vertico#551] [Q&A]
When using Vertico to select from a long selection list (larger than my minibuffer can display), I'd prefer TAB to page through candidates rather than choose the highlighted candidate.
Is this behavior possible via configuration?

Much appreciation!
Jeremy

**Comment by minad** [comment]
Sure, just change the keybindings appropriately. Please figure this out by consulting the Emacs and Vertico manuals. This is a rather basic problem.
